packages/wrangler/src/versions/view.ts

```diff
@@ -55,6 +62,11 @@ export async function versionsViewHandler(args: VersionsViewArgs) {
 
 	const version = await fetchVersion(accountId, workerName, args.versionId);
 
+	if (args.json) {
+		logRaw(JSON.stringify(version, null, 2));
```

Comment:
Why the `null, 2`? In general, I don't think we should be outputting prettified JSON

Reply:
Curious why you think we shouldn't output prettified JSON?

Seems to me like there's no benefit in _not_ prettifying it but if a user needed to debug their usage it would be helpful for it to be slightly more readable